pkg/rancher-desktop/assets/specs/command-api.yaml

```diff
@@ -316,6 +316,15 @@ components:
                     Use to disable flannel so you can install your own CNI.
                   x-rd-aliases:
                     - "flannel-enabled"
+            ingress:
+              type: object
+              properties:
+                localhostOnly:
+                  type: boolean
+                  x-rd-platforms: ['win32']
+                  x-rd-usage: >-
+                    Control whether K8s services are bound to localhost only or not.
```

Comment:
I think we need to elaborate on the other option too, perhaps something like:
> Control whether K8s services are bound to `127.0.0.1` only or `0.0.0.0`.

We may also need to document this to mention `nodePorts` and `LoadBalancer` service types are only applicable.